### PR TITLE
Fixed and updated Turkish translations

### PR DESCRIPTION
**Pull request by MetehanOzyurek**
* Translated news:
> 2023-02-28
> 2023-03-09
> 2023-04-06
* Fixed & updated Turkish translations for donate

### CODE REVIEW

**Review by biodranik — COMMENTED** [review]
@rtsisyk do we support Turkish via Weblate?

**Comment by MetehanOzyurek** [comment]
> @rtsisyk do we support Turkish via Weblate?

No, I prefer to translate on GitHub.

**Comment by MetehanOzyurek** [comment]
@biodranik 
https://hosted.weblate.org/projects/organicmaps/website/

File: content/news/2023-04-06/150/index.tr.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* API için haritadan nokta seçimi düzeltildi
+* Yokuş yukarı ve aşağı simgeleri daha net oklarla değiştirildi
+* Güç tasarrufu modundayken artık 3D yapılar devre dışı bırakılıyor
+* Yakınlaştırma butonları artık ayarlardan gizlenebilir
+* Gelişmiş açılış zamanı düzenleyicisi için hata işleme iyileştirildi
+* Yer imi listesi açıklaması artık görüntüleniyor
+*  F-Droid sürümünde yönlendirme yapmak artık harita indirmeyi durdurmuyor
+*  OsmLoginFragment'taki çökme sorunu düzeltildi
+*  Uyarı kutusu için koyu tema desteği eklendi
+*  Yer imlerini açarken artık alt sayfa kapatılmıyor
```
Comment (biodranik) [inline content/news/2023-04-06/150/index.tr.md:36]:
```suggestion
* F-Droid sürümünde yönlendirme yapmak artık harita indirmeyi durdurmuyor
* OsmLoginFragment'taki çökme sorunu düzeltildi
* Uyarı kutusu için koyu tema desteği eklendi
* Yer imlerini açarken artık alt sayfa kapatılmıyor
```

File: content/news/2023-02-28/143/index.tr.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: "OpenStreetMap ve Organic Maps, Dr. Bilgehan Çevik (https://www.openstreetmap.org/user/pedrito1414/diary/401061)'e Türkiye'de yardımcı oldu. "
```
Comment (biodranik) [inline content/news/2023-02-28/143/index.tr.md:2]: Please don't use markdown/links in titles.
```suggestion
title: "OpenStreetMap ve Organic Maps, Dr. Bilgehan Çevik'e Türkiye'de yardımcı oldu"
```

File: content/news/2023-04-06/150/index.tr.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Mutlu, Sağlıklı ve Huzur Dolu bir Ramazan dileriz"
+date: 2023-04-06T22:29:27+00:00
+slug: "you-can-easily-find-the-nearest-cannabis-store-with-the-latest-organic-maps-update"
```
Comment (biodranik) [inline content/news/2023-04-06/150/index.tr.md:4]: If you want, you may try to use some turkish letters for the slug.

**Review by biodranik — CHANGES_REQUESTED** [review]

**Review by biodranik — APPROVED** [review]
Thank you!

**Pull request merged**